### DISCUSSION TITLE
[Script request] Umbrel

### DISCUSSION THREAD

**Discussion by maximiliansr** [community-scripts/ProxmoxVE#412] [Request script]
### Application Name

Umbrel (OS)

### Website

https://umbrel.com/

### Description

Umbrel is a self-hosted operating system designed to make deploying personal cloud services easy and intuitive. With a user-friendly interface, Umbrel simplifies the management of apps like Bitcoin nodes, Nextcloud, Pi-hole, and other self-hosted services, making it ideal for both beginners and advanced users.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by gigi2006** [comment]
just wanted to ask, will there be umbrelOS or is it unlikely because of umbrel Home?

**Reply by FahadSalim** [reply]
There are instructions on the umbrel GitHub for Linux VM so hopefully, soon it’ll be integrated. There used to be a lxc before they went the umbrelOS way.

**Reply by tremor021** [reply]
Isn't this just a fancy frontend for docker?

**Reply by MickLesk** [reply]
Like runtipi

**Comment by MickLesk** [comment]
https://github.com/community-scripts/ProxmoxVED/issues/465#issuecomment-2909950559

**Comment by MickLesk** [comment]
Merged
